tests/framework/PostProcessors/SafestPoint/test_safest_point.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<Simulation verbosity="debug">
+<Simulation verbosity="silent">
```

Comment:
Should this be quiet instead of silent?

Reply:
change is not needed, otherwise, you need to update the revision info in the TestInfo

Reply:
The above comments is also valid for other tests except GA tests that got changes.